great-idea-website/css/index.css

```diff
@@ -63,4 +63,136 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+/* Whole Page */
+p {
+	font-family: 'Barriecito', cursive;
+}
+#container {
+	width:1000px;
+	margin:auto;
+}
+
+/* Links and Image  */
+nav * {
+	display:inline-block;
+	margin: auto 30px;
+	text-decoration:none;
+	color:lightgray;
+	font-size:14px;
+}
+
+/* Animations for Nav bar */
+
+nav a {
+	-webkit-animation: rotate .3s linear infinite alternate;
+}
+@-webkit-keyframes rotate {
+    from{
+        -webkit-transform: rotate(0deg);
+    }
+    to{
+        -webkit-transform: rotate(10deg);
+    }
+}
+/* Whole Nav Bar: Seperate navbar and header */
+nav {
+	margin-bottom: 50px;
+	background-color:rosybrown;
+	position:fixed;
+}
+
+/*  Main header */
+h1 {
+	font-size:70px;
+	margin-bottom:20px;
+}
+
+/* Header + Button */
+#title { 
+	display:inline-block;
+	width:50%;
+	margin:auto;
+	text-align:center;
+}
+/* Want everything to be centered vertically in nav and header */
+header * {
+	vertical-align:middle;
+}
+
+header {
+	margin-bottom:80px;
+}
+
+#start-button {
+	color:black;
+	text-decoration: none;
+	text-align:center;
+	border: 2px solid black;
+	padding:10px;
+	transition:.2s;
+}
+/* change color of starting button */
+#start-button:hover {
+	background-color:black;
+	color:white;
+}
+
+/* Main section */
+section { 
+	margin: 2% 0%;
+	border-top: 1px solid gray;
+	border-bottom: 1px solid gray;
+}
+
+/* Top half of info */
+.info {
+	display:inline-block;
+	width: 47%;
+	margin: 2% 2% 2% 0%;
+}
+
+/* Bottom half of info */
+.info-2 {
+	width:30%;
+	margin: 2% 2% 2% 0%;
+}
+/* Most bottom picture */
+section img {
+	width:100%;
+}
+
+/* Contact info spacing */
+address {
+	line-height:1;
+}
+
+/* Copyright */
+footer p {
+	margin:50px;
+	text-align:center;
+}
+
+#image-1 {
```

Comment:
I see where we are going with utilizing the `background-image` property to bring in some of these images that we are working with, but it looks like we have a slight hiccup with the pathing as the images probably show up when you are running things locally on your machine, but they do not show up if we spin up the index outside of your local environment. 

You can check that out here: 

http://htmlpreview.github.io/?https://github.com/jordowag/User-Interface/blob/db34a0e90a38f81064750567d6fd6008c38c7849/great-idea-website/index.html